tools/failures.py

```diff
@@ -90,6 +95,57 @@ def insert_in_database(to_insert, date=None):
     session.close()
 
 
+def reset_preseed():
+    data = session.query(JobPriorities.expires, JobPriorities.id)\
+                  .filter(JobPriorities.expires != None).all()
+
+    now = datetime.datetime.now()
+    for item in data:
+        try:
+            dv = datetime.datetime.strptime(item[0], "%Y-%M-%d")
+        except ValueError:
+            # TODO: consider updating column to have expires=None?
+            continue
+
+        # reset expire field if date is today or in the past
+        if dv.date() <= now.date():
+            conn = engine.connect()
+            statement = update(JobPriorities)\
+                          .where(JobPriorities.id == item[1])\
+                          .values(expires=None)
+            conn.execute(statement)
+
+
+def update_jobpriorities(to_insert, _priority, _timeout):
+    # to_insert is currently high priority, pri=1 jobs, all else are pri=5 jobs
```

Comment:
maybe we could make  to_insert looks like as a dict including job name and job priority (e.g   _**{job: job priority}**_ ), after we have 5 priorities job system.